edx_proctoring/callbacks.py

```diff
@@ -159,3 +160,24 @@ def get(self, request, attempt_code):  # pylint: disable=unused-argument
             },
             status=200
         )
+
+
+class StudentProctoredExamAttempt(APIView):
+    """
+    Test API, It will change the attempt status to 'submitted'
+    """
+
+    def get(self, request):  # pylint: disable=unused-argument
+        """
+        Returns the status of an exam attempt. Given that this is an unauthenticated
+        caller, we will only return the status string, no additional information
+        about the exam
+        """
+        return Response(
+            data={
+                # IMPORTANT: Don't add more information to this as it is an
+                # unauthenticated endpoint
+                'status': 'submitted',
+            },
+            status=200
+        )
```

Comment:
Hrm, can you explain this test endpoint? Is it for unit tests, or doing some manual tests? If it is for unit tests, I'd prefer some flag to expose this API as this will be a public API endpoint. I see that it doesn't really "do anything" (i.e. modify the database), but I'd rather not have this exposed.